pmd-core/src/main/java/net/sourceforge/pmd/renderers/XSLTRenderer.java

```diff
@@ -85,7 +85,10 @@ public void start() throws IOException {
         if (xslt == null) {
             throw new FileNotFoundException("Can't find XSLT file: " + this.xsltFilename);
         }
-        this.prepareTransformer(xslt);
+
+        try (InputStream stream = xslt) {
```

Comment:
This surprisingly seems to work for the rule - it initially complained about `xslt` needing to be closed.